test/integration/StakingNode.t.sol

```diff
@@ -1475,3 +1765,337 @@ contract StakingNodeWithdrawals is StakingNodeTestBase {
     }
 
 }
+
+contract StakingNodeOperatorSlashing is StakingNodeTestBase {
+    using stdStorage for StdStorage;
+    using BytesLib for bytes;
+    using SlashingLib for *;
+
+
+    address user = vm.addr(156_737);
+    uint40[] validatorIndices;
+
+    address avs;
+    address operator1 = address(0x9999);
+    address operator2 = address(0x8888);
+
+    uint256 nodeId;
+    IStakingNode stakingNodeInstance;
+    IAllocationManager allocationManager;
+    uint256 validatorCount = 2;
+    uint256 totalDepositedAmount;
+
+    function setUp() public override {
+        super.setUp();
+
+        avs = address(new MockAVS());
+
+        address[] memory operators = new address[](2);
+        operators[0] = operator1;
+        operators[1] = operator2;
+
+        for (uint256 i = 0; i < operators.length; i++) {
+            vm.prank(operators[i]);
+            delegationManager.registerAsOperator(address(0),0, "ipfs://some-ipfs-hash");
+        }
+
+        nodeId = createStakingNodes(1)[0];
+        stakingNodeInstance = stakingNodesManager.nodes(nodeId);
+
+        vm.prank(actors.admin.STAKING_NODES_DELEGATOR);
+        stakingNodeInstance.delegate(
+            operator1, ISignatureUtils.SignatureWithExpiry({signature: "", expiry: 0}), bytes32(0)
+        );
+
+        allocationManager = IAllocationManager(chainAddresses.eigenlayer.ALLOCATION_MANAGER_ADDRESS);
+        IStrategy[] memory strategies = new IStrategy[](1);
+        strategies[0] = IStrategy(stakingNodeInstance.beaconChainETHStrategy());
+        IAllocationManagerTypes.CreateSetParams[] memory createSetParams = new IAllocationManagerTypes.CreateSetParams[](1);
+        createSetParams[0] = IAllocationManagerTypes.CreateSetParams({
+            operatorSetId: 1,
+            strategies: strategies
+        });
+    
+        vm.prank(avs);
+        allocationManager.createOperatorSets(avs, createSetParams);
+
+        uint32 allocationConfigurationDelay = AllocationManagerStorage(address(allocationManager)).ALLOCATION_CONFIGURATION_DELAY();
```

Comment:
I created an IAllocationManagerExtended interface to get those constants, but your solution seems better :D